style.css

```diff
@@ -0,0 +1,90 @@
+.menu-icon {
+  align-self: center;
+  margin-right: 12px;
+  fill: #dbd8d7;
+}
+
+nav,
+main {
+  display: flex;
+  justify-content: space-between;
+  width: 375px;
+  background-color: antiquewhite;
+  margin: 0 auto;
+}
+
+main {
+  background-image: url("illustration.png");

```

Comment:
- [ ] **Applying the decoration background image to the headline section itself might be a better approach**

Currently, you have applied the background image to the main element. As you will add more content to the main element, it will grow and the background image will span across other elements as well. If you check the design, the background image is only applied to the headline section.